File: back/src/main/java/fr/uha/ensilan/concours/back/dto/message/NextQuestionMessage.java
package fr.uha.ensilan.concours.back.dto.message;

import fr.uha.ensilan.concours.back.dto.question.QuestionDto;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class NextQuestionMessage extends BaseMessage {
    QuestionDto question;

    public NextQuestionMessage() {
        super(MessageType.NEXT_QUESTION);
    }
}

File: front/src/app/core/domain/message/next-question.message.ts
import {MessageTypeEnum} from "@domain/message/message-type.enum";
import {QuestionDto} from "@domain/questions/question.dto";
import {BaseMessage} from "./base.message";

export class NextQuestionMessage extends BaseMessage {
  override readonly type = MessageTypeEnum.NEXT_QUESTION;
  question: QuestionDto;
}
